sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -1591,7 +1591,7 @@ def _add_argparse_args(cls, parser):
 class FlinkRunnerOptions(PipelineOptions):
 
   # These should stay in sync with gradle.properties.
-  PUBLISHED_FLINK_VERSIONS = ['1.12', '1.13', '1.14', '1.15', '1.16']
```

Comment:
note - this would essentially prevent python user using flink 1.17 for Beam 2.56.0, and not prevent user choosing 1.12,1.13. Consider update the known issue.

Reply:
Yeah, I sorta mentioned this, but I updated to be clearer/more explicit. Thanks